sql/system.go

```diff
@@ -40,9 +40,16 @@ const (
 	namespaceTableID  ID = 2
 	descriptorTableID ID = 3
 	usersTableID      ID = 4
+	zonesTableID      ID = 5
+
+	// NumUsedSystemIDs is only used in tests that need to know the
+	// number of system objects created at initialization.
+	// Make sure it is always set to the number of used system IDs.
+	// NOTE: it does not include RootNamespaceID since it has no
+	// corresponding KV data.
+	NumUsedSystemIDs = 5
```

Comment:
Why don't you make the IDs a separate const space with iota and add in a `maxTableID` entry so this is more automatic. You can then create an additional const which is `const NumusedSystemIDs = maxTableID - 1`